app/controllers/packages_controller.rb

```diff
@@ -61,6 +81,27 @@ def packages
     Package.configure(config)
   end
 
+  # Smooth over slicing out only the info RM API needs to create a
+  # custom package This also passes along the required provider id
+  def rm_api_package
+    find_provider
+
+    relevent_rm_api_data = package_params.slice(
+      :packageName,
+      :contentType,
+      :customCoverage
+    )
+
+    { vendor_id: @provider.vendorId }.merge(relevent_rm_api_data)
+  end
+
+  def find_provider
+    provider = Provider.configure(config)
+                 .all({search: config.customer_id })
```

Comment:
`{ search: config.customer_id }` (formatting)

Reply:
if rubocop didn't catch this we gotta take its badge & gun